nirps/qc_checks/check_coordinates.py

```diff
@@ -0,0 +1,137 @@
+# Pierrot, 2024-03-12
+
+from typing import Any, Dict, List, Tuple
+import numpy as np
+from astropy.coordinates import SkyCoord, AltAz, EarthLocation, Distance
+from astropy.table import Table
+from astropy.time import Time
+from astropy.io import fits
+import astropy.units as uu
+import glob
+import requests
+import pandas as pd
+from io import StringIO
+import warnings
+
+def ra_dec_to_alt_az(ra, dec, observer_latitude, observer_longitude, observation_time):
+    # Create a SkyCoord object with RA and DEC
+    coordinates = SkyCoord(ra=ra*uu.deg, dec=dec*uu.deg, frame='icrs')
+
+    # Set observer's location
+    observer_location = EarthLocation(observer_longitude*uu.deg, observer_latitude*uu.deg)
+
+    # Set observation time
+    observation_time = Time(observation_time)
+
+    # Transform coordinates to Altitude and Azimuth
+    altaz_coordinates = coordinates.transform_to(AltAz(obstime=observation_time, location=observer_location))
+
+    # Return Altitude and Azimuth
+    return altaz_coordinates.alt.deg*uu.deg, altaz_coordinates.az.deg*uu.deg
+
+def coords_check(fits_file, RA, DEC, tol=1/60, verbose=False): 
```

Comment:
variables like "RA", "DEC", "ALT", "AZ", "sin_error_ALT" etc should all be lowercase
- upper case is reserved for constants (i.e. MAIN_URL, PENDING_URL)
- CamelCase is reserved for class instances (i.e. SkyCoord, Time, AltAz)